src/System/i18n/locales/en-US/translation.json

```diff
@@ -74,7 +74,11 @@
       },
       "createAlert": {
         "description": "Get notifications for similar works"
-      }
+      },
+      "sellWithArtsy": "Want to sell a work by this artist? ",
```

Comment:
**question**: should we use a space at the end of the translation lines or is it better to add a space in the place where it will be rendered?

Reply:
Good question, I think it would be better if the space was in the place that the strings are rendered, will do that